packages/api/src/schema/schema.yaml

```diff
@@ -676,6 +676,19 @@ components:
         multistream:
           $ref: "#/components/schemas/stream/properties/multistream"
 
+    object-store-patch-payload:
+      type: object
+      additionalProperties: false
+      properties:
+        name:
+          $ref: "#/components/schemas/object-store/properties/name"
+        disabled:
```

Comment:
Nice, didn't know object stores and multistream targets had the `disabled` field, we could consider adding this as well to the signing keys for access control

Reply:
Yes definitely!! We need to be able to disable keys before deleting them, I thought we forgot to include that in the requirements docs.